salt/_modules/docker_registry.py

```diff
@@ -0,0 +1,85 @@
+'''
+Functions to interact with a Docker registry.
+'''
+
+import logging
+
+import salt.utils.json
+import salt.utils.path
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'docker_registry'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def get_image_manifest(image, tls_verify=True):
+    '''
+    Retrieve an image manifest from a repository.
+
+    .. note::
+
+       This uses the :command:`skopeo` command.
+
+    image
+        Name of the image (e.g. `localhost:5000/metalk8s/nginx:1.14.2`)
+    tls_verify : True
+        Verify TLS certificate of the repository. Set to `False` if the
```

Comment:
It's not clear if the `Set to False` is something automatically performed by the method or if it's an instruction for the developer.

Reply:
Instruction for the developer. Not sure why it'd be a call argument otherwise?